pages/blog/index.vue

```diff
@@ -367,6 +367,27 @@ const pageClasses = computed(() => {
     top: 0;
   }
 
+  /* this sets the image to fit the featured cards in safari too, this will be component change */
+  :deep(.ftva.block-highlight.is-vertical .image-container) {
```

Comment:
@tinuola if we put the nth-of-type on the block-hightlight, the first featured blog will have 1160/403 aspect ratio and the 2nd and 3rd will be 566/330
```
:deep(.ftva.block-highlight.is-vertical:nth-of-type(1) .image-container) {
    .image {
      aspect-ratio: 1160/403;

      .sizer {
        padding-bottom: calc(403/1160 * 100%) !important;
      }
    }
  }

  :deep(.ftva.block-highlight.is-vertical .image-container) {
    aspect-ratio: unset;

    .image {
      aspect-ratio: 566/330;

      .sizer {
        padding-bottom: calc(330/566 * 100%) !important;
      }
    }
  }
```